src/plugins/gauge/components/GaugeFormController.vue

```diff
@@ -27,6 +27,7 @@
         :class="model.cssClass"
     >
         <ToggleSwitch
+            :id="uuid()"
```

Comment:
What is the purpose of this ID? Where is this referenced?

Reply:
multiple toggle switch on form was causing issue,  I think I added id to check collisions (similar to :key in list)